prime-router/metadata/HL7/catchall/hl7/segments/PRT/PractitionerRole.yml

```diff
@@ -0,0 +1,40 @@
+# $schema: ./../../../../../json_schema/fhir/hl7-to-fhir-mapping-resource-template.json
+
+# TODO need message level conditions to be able to implement FHIR > HL7 mapping
+
+# This segment does not appear in NIST or RADx MARS
+
+resourceType: PractitionerRole
+
+id:
```

Comment:
Are `PRT/PractitionerRole` and `PRT/RelatedPerson` meant to be finished later? It looks like PractitionerRole at least is missing several fields.

Reply:
I started work on these before realizing the issue of needing message level conditions and think it would be better to defer work until those are available. I can add a comment to make that more clear.

Reply:
Yeah I agree, waiting to finalize these mappings once they can be used makes more sense. In that case lets just document either in the code or on the ticket what all should be revisited. Same applies to Observation-Location.